frontend/src/navbar.tsx

```diff
@@ -200,7 +200,9 @@ function SearchInput() {
         "search",
         patterns.length
           ? (pull: Pull) => {
-              return patterns.every((pattern) => pull.title.match(pattern));
+              const titleMatch = patterns.every((pattern) => pull.title.match(pattern));
+              const repoNameMatch = patterns.every((pattern) => pull.repo.match(pattern));
+              return titleMatch || repoNameMatch;
```

Comment:
I love it. Though now that I think about it, I think it would be more useable to do:

`patterns.every(pattern => title.match(patter) || repo.match(pattern))`

Because, as-is, this requires that your search query has to be entirely found in the repo name, or entirely found in the title.

Most searches are "search for this text anywhere in the searchable fields" i.e. operating as if the searchable fields were all concatenated together instead of treating each field as a separate document.

Reply:
While we are at this, would it be beneficial at all to include the pull number?

Reply:
![Screenshot 2023-04-27 092530](https://user-images.githubusercontent.com/95656772/234928354-25e8f253-7302-4c22-8ad2-3e77264c6d3f.jpg)

Reply:
I'm not sure I would use it, but it seems harmless.